pkg/gateway/middleware/authn.go

```diff
@@ -102,6 +108,10 @@ func isSkygearHeader(key string) bool {
 	return strings.HasPrefix(strings.ToLower(key), "x-skygear")
 }
 
+func isTenantConfigHeader(key string) bool {
+	return strings.EqualFold(key, corehttp.HeaderTenantConfig)
```

Comment:
Make sure x-skygear-request-id is kept too.

Reply:
refs https://github.com/SkygearIO/skygear-server/pull/1292/commits/df9a618582b3e20c0c96287fe97870dd7e0bf52a